### ISSUE TITLE
Update project's documentation

### ISSUE THREAD

**Issue by davegomez** [eventbrite/britecharts-react#146]
## Expected Behavior

To offer correct and complete documentation on how to use the library.

## Context | Current Behavior

The documentation offers a poor description on how to use the chart components as well on how to use the responsiveness helpers as described in the issue #144.

## Possible Solution

* Update the documentation with correct and tested code.
* Offer correct and easy to find dataset examples to test the charts.
* Offer CommonJS and ES5 examples on how to import and use the components.

**Comment by miglesiasEB** [comment]
Thanks @davegomez for the issue!

Yes, it makes sense to add docs about the responsive HOC and container.

For the test data, we could provide links to the fixtures in Britecharts for now, looking into a better solution in the future.

True, a tutorial on how to use the library with different options (CommonJS, ES5 and the CDN option) would be great too.

**Comment by dupski** [comment]
Yeah, I guess that the difficulty with open source being volunteer driven. Unfortunately I didn't have time to get to it that weekend and evidently no one else has :)

There's a discussion on the with responsiveness() helper here as I had the same issue:
https://github.com/eventbrite/britecharts-react/issues/144

HTH

**Comment by Golodhros** [comment]
Regarding loading the modules, I think this test project can help a bit on it:
https://github.com/Golodhros/britecharts-react-test-project/tree/master/src/donut
